src/test/java/com/github/rschmitt/dynamicobject/benchmark/DeserializationBenchmark.java

```diff
@@ -22,8 +22,8 @@ public class DeserializationBenchmark {
     @Test
     @Tag("benchmark")
     public void run() throws Exception {
-        DynamicObject.registerTag(StringFieldList.class, "string-field-list-tag");
-        DynamicObject.registerTag(StringField.class, "string-field-tag");
+        DynamicObject.registerTag(StringFieldList.class, "string-field-parent/string-field-list-tag");
```

Comment:
What's the purpose of changing this string?

Reply:
No really strong reasons, but my thought process was:
1) Benchmark will be closer to reality for any users of this class that use longer strings.
2) Shows that it can include characters like a "/" without issues.

Reply:
Should I remove the remaining changes in test code or only this line? The other changes are needed to properly test the "repeated registration" behavior (i.e. required to test what is being optimized with the src/main changes), but they are not breaking.